### DISCUSSION TITLE
Is it possible to put the env.yaml file anywhere and not just in $HOME/.sling

### DISCUSSION THREAD

**Discussion by ShahBinoy** [slingdata-io/sling-cli#93] [Q&A] [answered]
This is to allow just the definition to be placed along the code base, use relative paths or produce in different location in Docker. 
Forcing the file to be in `$HOME/.sling` is a bit too restrictive.

Or is it allowed/possible to pass in the `env.yaml` file as a CLI argument.

**Comment by flarco** [comment, marked as answer]
Yes, just provide env var `SLING_HOME_DIR`

**Comment by ShahBinoy** [comment]
Thank You.
